R/load-simulation.R

```diff
@@ -4,7 +4,7 @@
 #'
 #' @export
 loadSimulation <- function(pkmlSimulationFile) {
-  simulationLoader <- rClr::clrNew("OSPSuite.R.Services.SimulationLoader")
+  simulationLoader <- getNetTask("SimulationLoader")

```

Comment:
Just out of curiosity - does it have to be a separate SimulationLoader per simulation?

Reply:
It does not. Maintaining instances of services in memory may lead to unwanted leaks if said service is not implemented properly. In general, stateless objects can be discarded right away. No need to keep them in memory

Reply:
So maybe
1) Ether cash it like `containerTask`
2) or destroy it after loading the simulation `rm(simulationLoader)`?

Reply:
I would not cache it as it may lead to leaks. This is not often that such an object is created. We should only cache stuff when really really necessary

2. I am assuming this is destroyed automatically (e.g garbage collected) when the `loadSimulation` is completed no? What does rm do ? 
I certainly hope that local variable are destroyed when the function exit otherwise we have  bigger problems :)

Reply:
Yes, of course, got somehow confused... Yes it should get destroyed after existing the function.